ovis_bringup/config/ovis_controllers.yaml

```diff
@@ -0,0 +1,52 @@
+ovis:
+  # Publish all joint states -----------------------------------
+  joint_state_controller:
+    type: joint_state_controller/JointStateController
+    publish_rate: 100  
+
+# Trajectory Controllers ---------------------------------------
+  joint_trajectory_controller:
+    type: position_controllers/JointTrajectoryController
+    joints:
+      - arm_joint_1
+      - arm_joint_2
+      - arm_joint_3
+      - arm_joint_4
+      - arm_joint_5
+      - arm_joint_6
+    constraints:
+      goal_time: 0.5
+      stopped_velocity_tolerance: 0.2
+      arm_joint_1:
+        trajectory: 0.5
+        goal: 0.1
+      arm_joint_2:
+        trajectory: 0.5
+        goal: 0.1
+      arm_joint_3:
+        trajectory: 0.5
+        goal: 0.1
+      arm_joint_4:
+        trajectory: 0.5
+        goal: 0.1
+      arm_joint_5:
+        trajectory: 0.5
+        goal: 0.1
+      arm_joint_6:
+        trajectory: 0.5
+        goal: 0.1
+
+    state_publish_rate: 100
+
+
+# ovis/gazebo_ros_control:  
+#   pid_gains:
+#     arm_joint_1: {p: 100, d: 0.01, i: 0.01, i_clamp: 1}
+#     arm_joint_2: {p: 100, d: 0.01, i: 0.01, i_clamp: 1}
+#     arm_joint_3: {p: 100, d: 0.01, i: 0.01, i_clamp: 1}
+#     arm_joint_4: {p: 100, d: 0.01, i: 0.01, i_clamp: 1}
+#     arm_joint_5: {p: 100, d: 0.01, i: 0.01, i_clamp: 1}
+#     arm_joint_6: {p: 100, d: 0.01, i: 0.01, i_clamp: 1}
+      
+
```

Comment:
I don't think we need it, but right now when you launch moveit with gazebo, there are some errors because the PID gains haven't been specified. This code fix the errors, but since the gains haven't been properly adjusted, it doesn't work the way it should.

Reply:
I get it, there are still kinks to fix. Might want to add a comment detailing this with the keyword `TODO` while its still an issue. Extensions that highlight TODO in comments are pretty common and useful.